app/controllers/concerns/hyrax/homepage_controller_behavior.rb

```diff
@@ -35,14 +35,14 @@ def fetch_admin_sets
                                             .rows(5)
       response = repository.search(builder)
       response.documents
-    rescue Blacklight::Exceptions::ECONNREFUSED
+    rescue Blacklight::Exceptions::ECONNREFUSED, Blacklight::Exceptions::InvalidRequest
```

Comment:
Can the existing error go away if we use RSolr 2?  If so, can you add a comment about this.